packages/material/scss/pager/_theme.scss

```diff
@@ -0,0 +1,57 @@
+    @include exports("pager/theme") {
+    $pager-text: $header-text !default;
+    $pager-bg: $header-bg !default;
+    $pager-items-hover-bg: $list-item-hovered-bg !default;
+    $pager-numbers-text: $accent !default;
+    $pager-labels-text: rgba(0, 0, 0, .54) !default;
```

Comment:
Most of the code in this file can be omitted if you include the `_theme` file from theme-default. Most likely only the border-radius is specific to the material theme.